### PR TITLE
Added some explanation for the website build.

### PR DESCRIPTION
**Pull request by Confectrician**
I have added some explanation based on the discussion in #927.

Maybe you could share your opinion @davidgraeff and @openhab-5iver.

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by ghys** [comment]
Fyi- I just added some insight from my side on https://github.com/openhab/openhab-docs/pull/927.

**Comment by Confectrician** [comment]
> Fyi- I just added some insight from my side on #927.

Yes.
I will adapt the jenkins configuration based on your suggestion and improve/merge this information afterwards.

**Comment by Confectrician** [comment]
Just fyi:

Adapted configuration works as expected.
Tested it directly with this pr.

File: README.md [hunk 1]
```diff
@@ -74,6 +74,18 @@ The external content is updated by the following toolchain:
...
 
+#### Build triggers investigated
+
+There are two triggers available currently.
+The `merge docs` job is triggerd after something has been added to the documentation through this repository.
+The `gather external docs` job is started with a **succesful** build of the openhab-distribution.
+A succesful disribution build will include all of the latest changes that have been made to external sources like addons.
+So when a distribution build is succesful, we will trigger the gathering of all external sources.
+
+When one of these jobs is finished, we will then notify our website hosting service, to start a new website build.
```
Comment (openhab-5iver) [inline README.md]: > website hosting service, to start

website hosting service to start

File: README.md [hunk 2]
```diff
@@ -74,6 +74,18 @@ The external content is updated by the following toolchain:
...
+This recognized due to new commits in the final branch of this repository.
```
Comment (openhab-5iver) [inline README.md]: > This recognized

This is recognized

File: README.md [hunk 3]
```diff
@@ -74,6 +74,18 @@ The external content is updated by the following toolchain:
...
+The new build will include all the latest changes in the cods repository and in all external repositories.
```
Comment (openhab-5iver) [inline README.md]: > cods repository

code repository

**Comment by Confectrician** [comment]
gnah^^

will do the corrections directly.

**Comment by Confectrician** [comment]
> oops, I was too late!

Never too late for corrections. 🙂 
https://github.com/openhab/openhab-docs/commit/7209da63ddbee09fd48431a0b43282da1ac92505

**Pull request merged**